BlueShift-iOS-SDK/BlueShift.h

```diff
@@ -54,9 +54,10 @@ NS_ASSUME_NONNULL_BEGIN
 /// @param config BlueShiftConfig object
 + (void) initWithConfiguration:(BlueShiftConfig *)config;
 
-+ (void) autoIntegration;
++ (void) autoIntegration DEPRECATED_MSG_ATTRIBUTE("This method is deprecated, and will be replaced by the auto-integration using method swizzling. This method will be removed in future SDK versions.");
+
 - (void) setPushDelegate: (id) obj;
-- (void) setPushParamDelegate: (id) obj;
+- (void) setPushParamDelegate: (id) obj DEPRECATED_MSG_ATTRIBUTE("This method is deprecated, and will be removed in future SDK versions");
```

Comment:
can we also provide what is the alternative to the deprecated method? As we mentioned for autoIntegration method,